definitions.md

```diff
@@ -0,0 +1,66 @@
+# Continuous Delivery Definitions
+
+The purpose of this doc is to record definitions of important terminology in the Continuous
+Delivery space, based on [the canonical definitions of these terms](timeline.md).
+
+This is meant to be a living document: if you would like to propose updates to these definitions,
+[please open a PR](CONTRIBUTING.md).
+
+* [Continuous Delivery](#continuous-delivery)
+* [Continuous Deployoment](#continuous-deployment)
+* [Continuous Integration](#continuous-integration)
+* [CI/CD](#cicd)
+
+## Continuous Delivery
+
+**Continuous Delivery (CD)**: A software development practice in which teams release software changes to users safely,
+quickly and sustainably by:
+
+ 1. Proving that changes can be released at any time
+ 2. Automating release processes
```

Comment:
@tracymiranda i tried to capture both your more succinct definition AND these 2 points that i think are important but i can drop these 2 if you feel strongly 🙏